x/tx/decode/util.go

```diff
@@ -0,0 +1,15 @@
+package decode
+
+import (
+	"strings"
+
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+func MessageNameFromTypeURL(url string) protoreflect.FullName {
+	messagename := protoreflect.FullName(url)
+	if i := strings.LastIndexByte(url, '/'); i >= 0 {
+		messagename = messagename[i+len("/"):]
+	}
+	return messagename
+}
```

Comment:
I also encountered the need for this on 2-3 occasions, but ended up simply using `strings.TrimPrefix("/")`, because all SDK anys are assumed to start with `"/"`. If we do a utility function like this one, I would rather not put it here publicly in `x/tx/decode`. Maybe cosmos-proto?